ROOTFS/usr/bin/waggle-rpi-watchdog

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+import subprocess
+import time
+import logging
+
+logging.basicConfig(level=logging.INFO)
+
+last_connection_time = 0
+
+def update_systemd_watchdog():
+    try:
+        subprocess.check_call(["systemd-notify", "WATCHDOG=1"])
+    except subprocess.CalledProcessError:
+        logging.warning("skipping reset of systemd watchdog")
+
+
+def time_now():
+    return time.monotonic()
+
+
+def seconds_since(start):
+    return time.monotonic() - start
+
+def nx_connection_ok():
+    try:
+        return (
+            subprocess.check_output(["ping", "-c", "1", "10.31.81.1"])
+        )
+    except Exception:
+        return False
+
+
+# NOTE Revisit how much of the network stack we should restart. For now, I want to cover all
+# cases of wifi and modems and ssh tunnel issues.
+def restart_network_services():
```

Comment:
dont we want our recovery actions to be rebooting the RPi?  potentially logging into the network switch and having it pull and restore PoE power to the RPi?